datadog/internal/transport/custom_transport.go

```diff
@@ -4,22 +4,23 @@ import (
 	"bytes"
 	"context"
 	"io/ioutil"
+	"math"
 	"net/http"
 	"strconv"
 	"time"
 )
 
 var (
-	defaultHTTPRetryDuration = 5 * time.Second
-	defaultHTTPRetryTimeout  = 60 * time.Second
-	rateLimitResetHeader     = "X-Ratelimit-Reset"
+	defaultBackOffMultiplier float64 = 2
```

Comment:
nit; format indentation

Reply:
Thanks for the review. This was auto formatted using `goimports` which uses tabs for indentation